static/app/views/organizationGroupDetails/actions/index.tsx

```diff
@@ -268,7 +270,7 @@ class Actions extends React.Component<Props, State> {
           <ReviewAction onUpdate={this.onUpdate} disabled={!group.inbox || disabled} />
         </Tooltip>
         <DeleteAction
-          disabled={disabled}
+          disabled={disabled || hasEventsMemberAdmin}
```

Comment:
i think we should use the appropriate <Access> wrapper here

Reply:
maybe the `<Role>` component as well?

Reply:
I ended up just using the `<Access>` wrapper since it does what we need and allow users with the role of Admin and up see the delete button